apps/www/.storybook/main.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path")
+const { mergeConfig } = require("vite")
+module.exports = {
+  features: {
+    storyStoreV7: true,
+  },
+  stories: ["../components/**/*.stories.@(js|jsx|ts|tsx|mdx)"],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    /**
+     * Enable TailwindCSS JIT mode
+     * NOTE: fix Storybook issue with PostCSS@8
+     * @see https://github.com/storybookjs/storybook/issues/12668#issuecomment-773958085
+     */
+    {
+      name: "@storybook/addon-postcss",
+      options: {
+        // Note: Settings are picked up from postcss.config.cjs
+        postcssLoaderOptions: {
+          implementation: require("postcss"),
+        },
+      },
+    },
```

Comment:
```suggestion
    {
      name: "@storybook/addon-styling",
      options: {
        postCss: true,
      },
    },
```

In my current projects with Tailwind and Storybook, i'm using the `@storybook/addon-styling` plugin to get postcss working.  Maybe @Integrayshaun has another suggestion, but i know this works (also need to add the global styles to `.storybook/preview.tsx`)

Reply:
Hey @lloydrichards 👋 

In this case, since they’re using the vite builder, no options are needed to make this all work out of the box :) 

It can still be used for theme switching purposes though 💪